features/support/manual_helpers.rb

```diff
@@ -346,11 +346,8 @@ def get_id_for_manual(manual_title)
   def check_manual_has_organisation_slug(attributes, organisation_slug)
     # there's currently no mention of the manual's organisation on its page
     manual_id = get_id_for_manual(attributes.fetch(:title))
-    manual_services = ManualServiceRegistry.new
-
-    manual, _ = manual_services.show(manual_id).call
+    visit manual_path(manual_id)
```

Comment:
I think we might be able to just use `go_to_manual_page(attributes.fetch(:title))` now... And remove the method `get_id_for_manual` while we're at it, too, cause this is the only remaining use of it